python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -77,7 +76,10 @@ def __setattr__(self, attr, value):
         if attr == 'namePattern':
             if len(value.split(os.sep)) > 1:
                 this_dir = path.dirname(value)
-                super(LocalFile, self).__setattr__('localDir', this_dir)
+                if this_dir:
+                    super(LocalFile, self).__setattr__('localDir', this_dir)
+                elif path.isfile(path.join(os.getcwd(), path.basename(value))):
+                    super(LocalFile, self).__setattr__('localDir', os.getcwd())
```

Comment:
Probably. I just copied what was there before for sake of tracking changes when in development.